src/main/java/uk/ac/soton/itinnovation/security/systemmodeller/semantics/ModelObjectsHelper.java

```diff
@@ -330,7 +330,19 @@ public void deleteAssetFromCache(Asset asset, Model model) {
 		assetIDs.remove(uri);
 		assetUris.remove(id);
 	}
-	
+
+    public boolean cancelTask(String mId) {
```

Comment:
Is this method actually used anywhere? I suspect this was prototype code for cancelling the recommendations job?

Reply:
we do not this method, it is used to send and InterruptedException signal. We do not need it any more, should be removed.